### PR TITLE
fix: tests broken due to deprecated model

### PR DESCRIPTION
**Pull request by lfayoux**
`small` is no longer available on generate
`medium` is no longer available on classify
preset had to be changed since it was using `small` on generate

### CODE REVIEW

**Review by mkozakov — APPROVED** [review]

File: tests/test_classify.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class TestClassify(unittest.TestCase):
 
     def test_success(self):
-        prediction = co.classify(model='medium',
+        prediction = co.classify(model='small',
```
Comment (CoderHam) [inline tests/test_classify.py:12]: We have rules for rerouting of requests. We should understand if and how the re-routing broke. 

Approving but these changes shouldn't technically be needed

**Review by CoderHam — APPROVED** [review]

File: tests/test_classify.py (hunk 1 above)
Comment (CoderHam) [reply tests/test_classify.py:12]: PS: how come we do not need to do this for embed?

**Comment by lfayoux** [comment]
> should we remove `model` from the tests altogether, so that they never go stale?

we could, but we should still test that the model param works and preset request will also go stale anyways

File: tests/test_classify.py (hunk 1 above)
Comment (lfayoux) [reply tests/test_classify.py:12]: embed still has a small model that we're using to test

**Pull request merged**